qemu/tests/timerdevice_clock_drift_with_sleep.py

```diff
@@ -37,15 +38,16 @@ def verify_elapsed_time():
         output = session.cmd_output(get_time_cmd, timeout=timeout)
 
         times_list = output.splitlines()[1:]
-        times_list = [_ for _ in times_list if _ > 10.0 or _ < 11.0]
+        times_list = [_ for _ in times_list if float(_) < 10.0 or float(_) > 11.0]
 
         if times_list:
-            test.fail("Unexpected time drift found: Detail: '%s'" % output)
+            test.fail("Unexpected time drift found: Detail: '%s' \n timeslist: %s"
+                      % (output, times_list))
 
     error_context.context("Sync the host system time with ntp server",
                           logging.info)
-    process.system("yum install -y ntpdate; ntpdate clock.redhat.com",
-                   shell=True)
+    utils_package.package_install("ntpdate")
+    process.system("ntpdate clock.redhat.com")
```

Comment:
@xiangchunfu I am not sure, but looks like ntpdate is removed from RHEL8, could you help confirm what should be used instead here? Thanks.

Reply:
@vivianQizhu  You are right,  The latest RHEL8 tree has removed this nptdate package.  I also am finding solution.  once I found,  will update to this patch.

Reply:
@sitoliu please update this patch.  remove ntpdate.